enterprise/frontend/src/embedding-sdk/test/common-stories-utils.tsx

```diff
@@ -0,0 +1,51 @@
+import * as jose from "jose";
+import type { PropsWithChildren } from "react";
+
+import { MetabaseProvider, type SDKConfig } from "embedding-sdk";
+
+const METABASE_INSTANCE_URL =
+  (window as any).METABASE_INSTANCE_URL || "http://localhost:3000";
+const METABASE_JWT_SHARED_SECRET =
+  (window as any).JWT_SHARED_SECRET || "0".repeat(64);
+
+const secret = new TextEncoder().encode(METABASE_JWT_SHARED_SECRET);
+
+const DEFAULT_SDK_USER = {
+  firstName: "Robert",
+  lastName: "Tableton",
+  email: "normal@metabase.test",
+  password: "12341234",
+};
+
+const DEFAULT_CONFIG: SDKConfig = {
+  metabaseInstanceUrl: METABASE_INSTANCE_URL,
+  jwtProviderUri: `${METABASE_INSTANCE_URL}/sso/metabase`,
+  fetchRequestToken: async () => {
+    try {
+      const signedUserData = await new jose.SignJWT({
+        email: DEFAULT_SDK_USER.email,
+        first_name: DEFAULT_SDK_USER.firstName,
+        last_name: DEFAULT_SDK_USER.lastName,
```

Comment:
Two questions:

1. Couldn't we reuse the `normal` user from `cypress_data.js`?
2. What happens if first_name and/or last_name are not present? They are NOT the requirement in the main application - only email is.